dotcom-rendering/src/web/server/articleTemplate.ts

```diff
@@ -275,6 +277,7 @@ https://workforus.theguardian.com/careers/product-engineering/
 					${initTwitter}
 				</script>
 
+				${recipeMarkup}
```

Comment:
Could we move this tag into the body along with the low priority tags? I am assuming that the meta content is read by crawlers which will wait for the page to load?